src/util/dev/builder-cache.ts

```diff
@@ -201,10 +318,10 @@ export async function getBuilder(
     try {
       const mod = require(dest);
       const pkg = require(join(dest, 'package.json'));
-      builderWithPkg = Object.freeze({
-        builder: mod,
-        package: pkg
-      });
+      builderWithPkg = {
```

Comment:
Perhaps you should update the type `BuilderWithPackage` so that it has `builder: Readonly<Builder>`, etc

Reply:
Ahhh, I didn't know about that. Sounds good.